reindexer/src/main/scala/uk/ac/wellcome/platform/reindexer/modules/ReindexModule.scala

```diff
@@ -1,29 +1,56 @@
 package uk.ac.wellcome.platform.reindexer.modules
 
+import javax.inject.Singleton
+
 import akka.actor.ActorSystem
 import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import com.google.inject.Provides
 import com.twitter.inject.{Injector, TwitterModule}
-import uk.ac.wellcome.utils.GlobalExecutionContext.context
+import uk.ac.wellcome.models.{CalmTransformable, MiroTransformable}
+import uk.ac.wellcome.platform.reindexer.ServerMain
+import uk.ac.wellcome.platform.reindexer.services._
 import uk.ac.wellcome.utils.TryBackoff
 
-object ReindexModule extends TwitterModule  with TryBackoff {
+object ReindexModule extends TwitterModule with TryBackoff {
+
+  val targetTableName = flag[String](
+    name = "reindex.target.tableName",
+    help = "Reindex target table name")
+
+
+  @Singleton
+  @Provides
+  def providesMiroReindexTargetService(dynamoDBClient: AmazonDynamoDB): ReindexTargetService[MiroTransformable] =
```

Comment:
I don't think you need these provides methods. The inject annotation on their constructor should just work (I think?)